shared/actions/fs.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+import * as Constants from '../constants/fs'
+import * as FsGen from './fs-gen'
+import * as I from 'immutable'
+import * as RPCTypes from '../constants/types/rpc-gen'
+import * as Saga from '../util/saga'
+import * as Types from '../constants/types/fs'
+import {posix as Path} from 'path'
+
+import type {SagaGenerator} from '../constants/types/saga'
+
+export const fsPathToRpcPathString = (p: Types.Path): string =>
+  Types.pathToString(p).substring('/keybase'.length)
+
+function* folderList(action: FsGen.FolderListLoadPayload): SagaGenerator<any, any> {
+  if (action.payload.path === '/keybase') {
+    yield Saga.cancel()
```

Comment:
why do we need to cancel

Reply:
Same reason as above; RPC doesn't support listing /keybase but we'll get rid of this after the RPC changes is in.